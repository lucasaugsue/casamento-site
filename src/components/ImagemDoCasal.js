import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
    gridContainer: {
        display: "grid",
        position: "relative",
        gridTemplateColumns: "1fr auto auto auto 1fr",
    },
    gridItem: {
        textAlign: "center",
    },
    imageCasal: {
        float: "left",
        position: "absolute"
    },
    textoImage: {
        padding: "15px 15px 15px 15px",
        fontSize: "68px",
        fontWeight: 500,
        color:'white'
    }
}));

export default function ImagemDoCasal(){
    const classes = useStyles();

    return <div>
        <img className={classes.imageCasal} alt="" src="/fotodocasal.jpeg" />
        
        <div className={classes.gridContainer}>
            <div className={classes.gridItem}></div>
            <div className={classes.gridItem}>
                <div className={classes.textoImage}>Lucas</div>
            </div>
            <div className={classes.gridItem}>
                <div className={classes.textoImage}>&</div>
            </div>
            <div className={classes.gridItem}>
                <div className={classes.textoImage}>Vict√≥ria</div>
            </div>
            <div className={classes.gridItem}></div>
        </div>
    </div> 
}
