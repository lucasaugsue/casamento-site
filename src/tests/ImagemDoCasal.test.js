import renderer from 'react-test-renderer';
import ImagemDoCasal from '../screens/ImagemDoCasal';

test('Renderiza ImagemDoCasal corretamente', async () => {
    const tree = renderer
      .create(<ImagemDoCasal/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });