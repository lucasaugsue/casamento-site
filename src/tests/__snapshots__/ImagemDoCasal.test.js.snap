// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderiza ImagemDoCasal corretamente 1`] = `
<section
  className="container"
  id="inicio"
>
  <div
    className="image"
  />
  <div
    className="divText"
  >
    <div
      className="text"
    >
      <div
        className="containerRotateScroling"
      >
        <div
          style={
            {
              "transform": "rotate(10deg)",
            }
          }
        >
          L
        </div>
        <div
          style={
            {
              "transform": "rotate(20deg)",
            }
          }
        >
          u
        </div>
        <div
          style={
            {
              "transform": "rotate(30deg)",
            }
          }
        >
          c
        </div>
        <div
          style={
            {
              "transform": "rotate(40deg)",
            }
          }
        >
          a
        </div>
        <div
          style={
            {
              "transform": "rotate(50deg)",
            }
          }
        >
          s
        </div>
        <div
          style={
            {
              "transform": "rotate(60deg)",
            }
          }
        >
           
        </div>
        <div
          style={
            {
              "transform": "rotate(70deg)",
            }
          }
        >
          &
        </div>
        <div
          style={
            {
              "transform": "rotate(80deg)",
            }
          }
        >
           
        </div>
        <div
          style={
            {
              "transform": "rotate(90deg)",
            }
          }
        >
          V
        </div>
        <div
          style={
            {
              "transform": "rotate(100deg)",
            }
          }
        >
          i
        </div>
        <div
          style={
            {
              "transform": "rotate(110deg)",
            }
          }
        >
          c
        </div>
        <div
          style={
            {
              "transform": "rotate(120deg)",
            }
          }
        >
          t
        </div>
        <div
          style={
            {
              "transform": "rotate(130deg)",
            }
          }
        >
          รณ
        </div>
        <div
          style={
            {
              "transform": "rotate(140deg)",
            }
          }
        >
          r
        </div>
        <div
          style={
            {
              "transform": "rotate(150deg)",
            }
          }
        >
          i
        </div>
        <div
          style={
            {
              "transform": "rotate(160deg)",
            }
          }
        >
          a
        </div>
      </div>
    </div>
  </div>
</section>
`;
